app/views/directives/_service-binding.html

```diff
@@ -48,4 +48,18 @@
       View Secret
     </a>
   </div>
+  <div class="service-binding-parameters" ng-if="$ctrl.bindParameterSchema.properties && (true || !($ctrl.binding | isBindingFailed) && ($ctrl.binding | isBindingReady))">
```

Comment:
Is there a reason not to show parameter for failed bindings? It might help the user debug.